docs/entity_registry_index.md

```diff
@@ -3,7 +3,7 @@ title: Entity Registry
 sidebar_label: Introduction
 ---
 
-The entity registry is a registry where Home Assistant keeps track of entities. Any entity that is added to Home Assistant and has a unique ID will be registered in the registry.
+The entity registry is a registry where Home Assistant keeps track of entities. Any entity that is added to Home Assistant and has a unique ID (the `unique_id` attribute) will be registered in the registry.
```

Comment:
The unique ID attribute being called `unique_id` is uh, obvious? I don't know if that adds anything. You can reword it maybe 
```suggestion
The entity registry is a registry where Home Assistant keeps track of entities. Any entity that is added to Home Assistant which specifies the [`unique_id` attribute](https://developers.home-assistant.io/docs/core/entity#generic-properties) will be registered in the registry.
```

Reply:
@balloob - No, that's the whole point to documentation - explaining what's not obvious to new developers.
I thought I already had a unique ID because my entity_ids were unique in Home Assistant. This page doesn't currently say how to assign a unique ID. What if the property required were `uniqueid` or `entity_id` or... ? I don't know until someone tells me.
The link to the generic properties is great, because there was no connection I could see in the docs between these pages. Nice.